sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Snippets/Sample07_MockingEventProcessorHandlersTest.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Processor;
+using Moq;
+using NUnit.Framework;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Azure.Messaging.EventHubs.Tests.Snippets
+{
+    [TestFixture]
+    public class Sample07_MockingEventProcessorHandlersTest
+    {
+        /// <summary>
+        ///   Performs basic unit test validation of the contained snippet.
+        /// </summary>
+        ///
+        [Test]
+        public async Task MockingProccessEventHandler()
+        {
+            #region Snippet:EventHubs_Processor_Sample07_MockingProcessEventArgs
+
+            var mockPartitionId = "<< Event Hub Partion Id >>";
+            var mockPartitionContext = EventHubsModelFactory.PartitionContext(mockPartitionId);
+            var mockEventBody = new BinaryData("This is a sample event body");
+            var mockEventData = EventHubsModelFactory.EventData(mockEventBody);
+
+            var mockProcessEventArgs = new ProcessEventArgs(mockPartitionContext, mockEventData, updateToken => Task.CompletedTask, CancellationToken.None);
```

Comment:
Once we get the overall form to the point that we're comfortable, we'll want to keep an eye on line length;  the samples get read on GitHub more often than downloaded.  We've had some strong feedback to earlier samples that we needed to make that we fit into the GitHub width to keep things readable.

Reply:
sounds reasonable, so I should probably push updateToken to next line then...